support/mender-inventory-update-modules

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+#
+# Returns an the update-modules currently installed on the system
+# For more information on update-modules see:
+# https://hub.mender.io/c/update-modules/13
+#
+
+set -e
+
+FOUND=0
```

Comment:
This I'm not sure if we need or not?
Like, do we reset the inventory on a fresh rootfs install?

Reply:
No, we don't. So I think it's a good idea to have this.

Reply:
I guess that then, maybe we should bring this idea to the rest of the scripts?
@lluiscampos what do you think?

Reply:
Good idea. I agree :+1:

Reply:
I wen't through the other `inventory` scripts, and they actually seem quite good. The all report some zero value, in case the expected values are not found. This means that they behave like we expect them to.